Lombiq.BaseTheme/package.json

```diff
@@ -1,10 +1,27 @@
 {
   "private": true,
+  "dependencies": {
+    "bootstrap": "^5.2.3"
```

Comment:
@DAud-IcI Is it generally a problem to update Bootstrap here? Because they've changed some things which break the current UI tests, but I'd be willing to fix that, because the update also fixed a completely different problem with `autoprefixer`.

Reply:
How are UI tests broken? If you update Bootstrap to a version with breaking changes (where you have to change the HTML to fix something after the upgrade), that could be a problem for other modules that use Bootstrap and don't know about Lombiq.BaseTheme, e.g. Lombiq.DataTables.

Reply:
Thanks, I wasn't quite sure about how the bootstrap version in BaseTheme affects other modules.

But your explanation is right on point:
> you have to change the HTML to fix something after the upgrade

And that's totally out of scope. Way out.